HELENA_ROG.MD

```diff
@@ -0,0 +1,16 @@
+
+        ◦ My name: Helena Rog (just Helen :)
+          
+        ◦ My favourite sport and why I like it: hiking and trekking in the mountains. Last time I climbed Czantoria Wielka/ Velká Čantoryje which is a cross border Poland/Czech mountain. I like mountaineering because it gives me a sense of accomplishment and conquer. Besides, I like tennis and badminton and in good weather I play with my son. I like it because it improves concentration and focus.
+          
+        ◦ My favourite food and why it's my favourite: Trying to thing hard but I don’t have a favourite food. 
+          
+        ◦ A link to a video of my favourite music: I like all sorts of music but mostly orchestral and classical and to some extent- power metal. I tend to steer towards instrumental music, lyrics are not that important to me but if there are any, they better be meaningful. On a normal basis I binge listen to all albums by Two Steps From Hell and Audiomachine.  I also like music by Hanz Zimmer , Zack Hemsey, Max Richter, Sabaton, Rodrigo y Gabriela.
```

Comment:
Oh, I like Hans Zimmer, too :) I also like Thomas Newman and Yann Tiersen, but I think they are somewhat of a different style.